jenkins/build.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###
+### NOTE: Replacement for hack/jenkins/build.sh ???
```

Comment:
Looking at build.sh, it sure looks like a candidate to move into test-infra anyway, no?
It's very jenkins specific and very test-infra specific and no doubt only lives in kubernetes proper for historical reasons.  Adding this modified new version here in the hopes I can help make this the first of many scripts moved out of hack/jenkins over to test-infra?